### ISSUE TITLE
Error in dist/textAngular.min.js

### ISSUE THREAD

**Issue by varmint09** [fraywing/textAngular#1058]
Hi,

I am using gulp as part of my build process, and the min file is throwing an error;

/node_modules/textAngular/dist/textAngular.min.js: Unexpected token (9:27399)

Also, why is there demo folders in the NPM package, surely thats wasteful? I suggest they get removed and remain in GitHub but not NPM.

**Comment by bencpeters** [comment]
+1

**Comment by pbassut** [comment]
There shouldn't be a demo folder in the npm package. As of now, I don't have permission to fix this. But I'll fix it as soon as I get it.

**Comment by pbassut** [comment]
About the error. Could you unminify that on chrome and tell me the part of code that's causing the problem?
That's odd. It doesn't happen when using grunt.

**Comment by wellcaffeinated** [comment]
+1
Same problem when using browserify

> error: Parsing file (...)/node_modules/textangular/dist/textAngular.min.js: Unexpected token (9:27399)

**Comment by varmint09** [comment]
This also seems to be referenced here; #1068 so its quite a common bug, and I would imagine that gulp usage is growing far faster than grunt.

**Comment by pbassut** [comment]
@varmint09 Yeah, I know. I asked for a sample project but the OP didn't provide it. 
Could you set it up?

**Comment by varmint09** [comment]
@pbassut I use this as my boilerplate; https://github.com/jakemmarsh/angularjs-gulp-browserify-boilerplate

If you add your usual setup to this, you should reproduce our issues... its a gulp-browserify boilerplate.

I would share mine, but its considerably more complex than that one, and would muddy rather than clear the waters.

Thanks!

**Comment by zacol** [comment]
+1

**Comment by partynetwork** [comment]
+1

**Comment by timosadchiy** [comment]
+1

**Comment by vnikiti** [comment]
As a work around, below is code that I used to get around this problem.
`window.rangy = require('rangy/lib/rangy-core');
require('rangy/lib/rangy-selectionsaverestore');
var angularSanitize = require('angular-sanitize');
require('textangular/dist/textAngular-sanitize.min');
window.taTools = {};
require('textangular/dist/textAngularSetup');
require('textangular/dist/textAngular');`

**Comment by JoelParke** [comment]
<p>Is this still an issue with 1.5.16? since I don't use browserfy currently.  Or is the work around @vnikiti added sufficient?</p>
